#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_launch-server_tool_debianization.dpatch by Andres Mejia <mcitadel@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to set proper AA Root Directory and AA Binary Name in launch-server script.
## This patch will work provided that the source tree was checked out to a directory named alienarena2007.

@DPATCH@
diff -urNad alienarena2007~/Tools/LinuxScripts/launch-server alienarena2007/Tools/LinuxScripts/launch-server
+++ alienarena2007/Tools/LinuxScripts/launch-server	2007-05-12 00:49:30.000000000 -0400
@@ -17,11 +17,11 @@
 #ulimit -c unlimited
 
 ### Set your default AA Root Directory
-aadir="/home/aa/alienarena2007"
+aadir="/usr/share/games/alienarena2007"
 
 ### Set your AA Binary Name or command string ($1 is the argument
 ### passed to the script)
-aabin="crded +set game arena +exec $1"
+aabin="/usr/lib/games/alienarena2007/crded +set game arena +exec $1"
 
 ########## you probably don't need to change anything below here 
 ##########
